src/test/java/com/epam/reportportal/junit/junit/UnitTestRunner.java

```diff
@@ -0,0 +1,22 @@
+package com.epam.reportportal.junit.junit;
```

Comment:
This runner is used to attach the ReportPortalListener so that test execution events are actually transmitted to the backend server.